### PR TITLE
add diamond post

### PR DESCRIPTION
**Pull request by simonbray**

### CODE REVIEW

**Comment by erasche** [comment]
Can one of the admins verify this patch?

File: _posts/2019-05-28-diamond.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+site: freiburg
+title: "Visit from Macromolecular Crystallography Group"
+tags: ???
```
Comment (simonbray) [inline _posts/2019-05-28-diamond.md]: no idea what should go here

File: _posts/2019-05-28-diamond.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+supporters:
+ - eosc
+---
```
Comment (simonbray) [inline _posts/2019-05-28-diamond.md]: is this right? should I also mention them in the text?

**Comment by simonbray** [comment]
Is this more or less what you had in mind @bgruening?

File: _posts/2019-05-28-diamond.md (hunk 1 above)
Comment (erasche) [reply _posts/2019-05-28-diamond.md]: You can grep for `tags:` in the post directory to find out a list of options. Probably:

```
tags: [tools]
```

Comment (simonbray) [reply _posts/2019-05-28-diamond.md]: The problem was that none of the tags really seemed to fit. But I guess tools works as well as anything else, thanks.

Comment (erasche) [reply _posts/2019-05-28-diamond.md]: Is the tool in galaxy? Can you link to it? Tags are whatever you want them to be. a couple have special icons, but if you think there's a more appropriate tag, just apply it. E.g. `visit` could be fine, or `collaboration`

File: _posts/2019-05-28-diamond.md (hunk 2 above)
Comment (bgruening) [reply _posts/2019-05-28-diamond.md]: Yes, but maybe we need to add EOSC to the website-framework. Meaning adding a logo etc to the website.

**Comment by bgruening** [comment]
@simonbray yes. I dated the blog-post back and added a few more links.

File: _posts/2019-05-28-diamond.md (hunk 1 above)
Comment (simonbray) [reply _posts/2019-05-28-diamond.md]: Ah okay. Then `collaboration` is probably indeed better.

File: _posts/2019-04-23-diamond.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+---
+site: freiburg
+title: "Visit from Macromolecular Crystallography Group"
```
Comment (tdudgeon) [inline _posts/2019-04-23-diamond.md]:
```suggestion
title: "Visit from Diamond's Macromolecular Crystallography Group"
```

File: _posts/2019-04-23-diamond.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+tags: [tools]
+supporters:
+ - eosc
+---
+
+Recently, we have started working with the (Macromolecular Crystallography group)[https://www.diamond.ac.uk/Instruments/Mx.html], which is based at the [Diamond synchrotron](https://www.diamond.ac.uk) in Didcot, UK. We have a common interest in enabling reproducible, user-friendly data analysis in cheminformatics and computational chemistry. They are developing (Squonk)[https://squonk.it], a computational notebook with similar aims to the [ChemicalToolbox](http://cheminformatics.usegalaxy.eu) we are working on.
```
Comment (tdudgeon) [inline _posts/2019-04-23-diamond.md]:
```suggestion
Recently, we have started working with the (Macromolecular Crystallography group)[https://www.diamond.ac.uk/Instruments/Mx.html], which is based at the [Diamond synchrotron](https://www.diamond.ac.uk) in Didcot, UK. We have a common interest in enabling reproducible, user-friendly data analysis in cheminformatics and computational chemistry. As part of their [XChem project](https://www.diamond.ac.uk/Instruments/Mx/Fragment-Screening.html) they are developing computational workflows to assist in designing follow-up compounds from fragment screening hits.

Diamond is working with [Informatics Matters](https://www.informaticsmatters.com/) to this aim, and their (Squonk Computational Notebook)[https://squonk.it], a computational notebook with similar aims to the [ChemicalToolbox](http://cheminformatics.usegalaxy.eu) we are working on, is one part of the end user facing tools. We were interested in investigating how Squonk and Galaxy could be integrated. 
```

File: _posts/2019-04-23-diamond.md [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+
+Last week, [Tim Dudgeon](https://www.informaticsmatters.com) and [Rachael Skyner](https://www.diamond.ac.uk/Instruments/Mx/Fragment-Screening/XChem-team/Skyner.html) visited Freiburg to find out more about Galaxy. We spent two days discussing and together developed a new Galaxy tool which can be used as part of a protein-ligand docking workflow. Eventually, we hope to create a common base of tools for compuational chemistry that can be utilized by both of our platforms.
```
Comment (tdudgeon) [inline _posts/2019-04-23-diamond.md]:
```suggestion
Recently, [Tim Dudgeon](https://www.linkedin.com/in/timdudgeon/) from Informatics Matters and [Rachael Skyner](https://www.diamond.ac.uk/Instruments/Mx/Fragment-Screening/XChem-team/Skyner.html) from Diamond visited Freiburg to find out more about Galaxy. We spent two days discussing and together developed a new Galaxy tool which can be used as part of a protein-ligand docking workflow. The aim is to get a Galaxy instance running at Diamond in the near future, and eventually, we hope to create a common base of tools for compuational chemistry that can be utilized by both of our platforms.
```

**Comment by simonbray** [comment]
Thanks @tdudgeon, I've added your suggestions.

**Comment by bgruening** [comment]
Thanks all, it's online https://galaxyproject.eu/posts/2019/04/23/diamond/ now.

You can still change it if you like via a PR!

**Pull request merged**